src/archanaNJun20/ArmstrongNumber.java

```diff
@@ -0,0 +1,22 @@
+package archanaNJun20;
+
+public class ArmstrongNumber {
+	static void printArmstrongNumber(int number){
+		int oNumber=number;
+		int lastdigit=0;
+		int cubeAddition=0;
+		while(number>0){
+			lastdigit=number%10;
+			number=number/10;
+			cubeAddition=cubeAddition+(lastdigit*lastdigit*lastdigit);
```

Comment:
Here you should use math function like Math.pow(number%10, 3). 
So no need to create int lastdigit=0; this variable.

Reply:
The code is updated based on comments.